src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinStr = '';
```

Comment:
```suggestion
    let joinedStr = '';
```

Reply:
join - action, joined - result of action, let's create more meaningful variables names